modules/ROOT/pages/single-sign-on-with-saml.adoc

```diff
@@ -310,19 +310,19 @@ If the IdP declares a redirect/target URL, it might override the target URL set
 
 == Configure logout behaviour
 
-If your Bonita platform is configured to manage authentication over SAML, when users log out of Bonita Portal, they do not log out of the SAML Identity Provider (IdP).
+If your Bonita platform is configured to manage authentication over SAML, when users log out of Bonita Runtime, they do not log out of the SAML Identity Provider (IdP).
 Therefore they are not logged out of all applications that are using the IdP.
 To avoid this, you have two options :
 
-=== Hide the logout button of the portal
+=== Hide the logout button of the Bonita Layout
 
 This is the most commonly used solution. Users are logged in as long as they don't close their web browser (unless their session times out).
```

Comment:
Users are logged in as long as they don't close their web browser (unless their session times out). => Users are logged in as long as they don't close their web browser and the session does not time out.

Reply:
Done previously